packages.json

```diff
@@ -23487,5 +23487,22 @@
     "description": "Youtube data extractor",
     "license": "MIT",
     "web": "https://github.com/thisago/ytextractor"
+  },
+  {
+    "name": "notcurses",
+    "url": "https://github.com/michaelsbradleyjr/nim-notcurses",
+    "method": "git",
+    "tags": [
+      "TUI",
+      "character graphics",
```

Comment:
no spaces in tags plz

Reply:
Fixe that, and I shortened the description since shorter descriptions seem to be favored.